scripts/src/main/resources/scripts/functions

```diff
@@ -357,7 +357,7 @@ function doExtract() {
   if [ "${filename_base/*\./.}" = ".tar" ] || [ "${ext}" = ".tar" ] || [ "${ext}" = ".tgz" ] || [ "${ext}" = ".tbz2" ]
   then
     doRunCommand "tar xf '${1}' -C '${target_dir}'"
-  elif [ "${ext}" = ".zip" ]
+  elif [ "${ext}" = ".zip" ] || [ "${ext}" = ".jar" ]
```

Comment:
I have seen strange effects where JAR files needed to be extracted with the `jar` tool instead of `unzip` that was generating checksum errors in some edge-cases.